xml/depl_cisco_opflex.xml

```diff
@@ -0,0 +1,686 @@
+    <step>
+    <para>
+        Enable <literal>No Link and IP Setup</literal>.
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+       Go to the General tab and set the MTU to 1600.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+       Press F10 to continue.
+    </para>
+   </step> 
+</procedure>
+
+
+<para>
+    You should be back at the first screen at <literal>YAST | System | Network Settings</literal>. Press F3 (Add) to open the Hardware Dialog. 
+</para>
+
+  <procedure>
+   <step>
+    <para>
+      Under <literal>Device Type</literal> select VLAN.
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+        Set the <literal>Configuration Name</literal> to match what has been defined in the ACI fabric. In the example network that is <literal>vlan.4093</literal>.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+        Press F10 (Next) to return to the <literal>Network Card Setup | Address screen.</literal>
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+       Select the Ethernet device that is connected to the ACI fabric  for <literal>Real Interface for VLAN</literal>.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+       Enter your VLAN ID, which in the example network is 4093.
+    </para>    
+   </step> 
+    
+   <step>
+    <para>
+      Enable <literal>Dynamic Address - DHCP</literal>.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+      Change to the General tab and set the MTU to 1600.
+    </para>
+   </step> 
+
+    <step>
+    <para>
+      Press F10 to save changes for new VLAN device, and press F10 again to exit Network Settings.
+    </para>
+   </step>    
+</procedure>
+
+
+<para>
+   Now that you have made these changes, re-enter Network Settings and configure routing. 
+</para>
+
+<procedure>
+   <step>
+    <para>
+     Go back into Network Settings (Click Continue when you see the Chef client warning).
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+        Change to the Routing tab and add these entries to the Routing Table:
+        
+        <command>Destination     224.0.0.0
+        Genmask         240.0.0.0
+        Gateway         0.0.0.0
+        Device          vlan.4093 </command>(Select the VLAN device created in the previous steps)
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+     Press F10 to save and exit Network Settings, then press F9 to exit Yast.
+    </para>
+   </step> 
+ </procedure>
+</sect1>
+
+<sect1 xml:id="sec.connectivity">
+  <title>Confirm Correct Connectivity</title>
+
+  <para>
+      Edit the <filename>/etc/dhclient.conf</filename> file on all compute nodes to verify that <literal>send dhcp-client-identifier</literal> has the MAC address of the NIC on the ACI fabric. (Run <command>ip addr show</command> to find the MAC address.) It should look like this, uncommented, with <literal>1:</literal> prepended to your MAC address, followed by a semi-colon, like this example: 
+   </para>
+   
+    <para> 
+        <literal>  
```

Comment:
I would rather use &lt;screen>, because this is a single command in a block element - execatly what we are using &lt;screen> for